worker/fbs/notification.fbs

```diff
@@ -55,6 +55,7 @@ union Body {
     DataProducer_SendNotification: FBS.DataProducer.SendNotification,
 
     // Notifications from worker.
+    PlainTransport: FBS.PlainTransport.NotificationWrapper,
```

Comment:
This type (Notification::Body) will only have the respective namespaces (PlainTransport, DirectTransport, Producer, Consumer, etc), not the specific notifications anymore.